src/libraries/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -17,20 +17,19 @@ System.Runtime.CompilerServices.Unsafe</PackageDescription>
     <ExtraMacros>#define netcoreapp</ExtraMacros>
     <CoreAssembly>System.Runtime</CoreAssembly>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+  <PropertyGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETStandard'">
     <CoreAssembly>netstandard</CoreAssembly>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetFramework)' == 'net461'">
+  <PropertyGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">
     <CoreAssembly>mscorlib</CoreAssembly>
-    <ExtraMacros>#define $(TargetFramework)</ExtraMacros>
```

Comment:
why is this being removed ?

Reply:
The macro wasn't used.

Reply:
How isn't it being used on line 62?

Reply:
L62 adds the #define to the generated header file content. The System.Runtime.CompilerServices.Unsafe.il file which imports the header file doesn't use that define.

Reply:
Then you should also be able to remove it on line 17.

Reply:
And remove line 62

Reply:
Oh, I see.  In the .il file, it uses `#ifdef netcoreapp` and `#else`. It doesn't use any other define.